[Pytorch Edge] Update Black Box Api with operator versioning

JacobSzwejbka opened the pull request:
Differential Revision: D32573603

cccclai reviewed (commented):
LGTM. Can we have cover it in the unit test?

JacobSzwejbka committed:
[Pytorch Edge] Get Operator Version from model file (#68677)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/68677

Using in compatibility apis. Luckily the stream reader kinda just does this already so mostly just create a wrapper in our compatibility files

Test Plan: ci

Differential Revision: D32573132

fbshipit-source-id: 74bf54bf51d2d54278bbce23ec1b7b897e0a8170

JacobSzwejbka committed:
[Pytorch Edge] Update Black Box Api with operator versioning (#68678)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/68678

Test Plan: Ill update the unit test before land

Reviewed By: cccclai

Differential Revision: D32573603

fbshipit-source-id: dfe17a26e35c580156e01b1d5f44b6e80e882f42
